CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing Guidelines
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
```

Comment:
Check with mdlint.

Reply:
Do we want to have a sentence on each line or stick to a certain line length?

Reply:
The ROS style is one sentence per line and 80 cols I think we could go with that.